lib/core/src/getRodsEnv.cpp

```diff
@@ -142,13 +142,14 @@ extern "C" {
         createRodsEnvDefaults( &rodsEnvArg );
     }
 
-    static int capture_string_property(const std::string& _key, char* _val, std::size_t _val_size)
+    int capture_string_property(const char* _key, char* _val, size_t _val_size)
     {
+        std::string key{_key};
```

Comment:
`_key` needs to be checked for `nullptr` before constructing the `std::string`.

Reply:
Done